### ISSUE TITLE
npm audit found vulnerabilities

### ISSUE THREAD

**Issue by temmiland** [tomxpcvx/react-fileicons#46]
```
# npm audit report

braces  <3.0.3
Severity: high
Uncontrolled resource consumption in braces - https://github.com/advisories/GHSA-grv7-fg5c-xmjg
fix available via `npm audit fix`
node_modules/braces

1 high severity vulnerability

To address all issues, run:
  npm audit fix

```

**Comment by temmiland** [comment]
```
# npm audit report

braces  <3.0.3
Severity: high
Uncontrolled resource consumption in braces - https://github.com/advisories/GHSA-grv7-fg5c-xmjg
fix available via `npm audit fix`
node_modules/braces

1 high severity vulnerability

To address all issues, run:
  npm audit fix

```

**Comment by temmiland** [comment]
Fixed with [3000f96](https://github.com/temmiland/react-fileicons/commit/3000f96f92e884389736353716a50a56328cff92)
